README.md

```diff
@@ -20,6 +20,20 @@ minutiae as the TSC delegates most of its responsibilities to other projects and
 Every Top Level Project not currently incubating can appoint someone to the TSC who they elect
 at their own discretion.
 
```

Comment:
Yeah, that's the problem with having it below. It just seems a bit redundant to add another header?

If anything, I think "Current members" makes the most sense as the previous header already states "TSC".